pages/freeregistration.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import type { NextPage } from "next";
+import homeStyles from "../styles/Home.module.css";
+import styles from "../styles/discount.module.css";
+import DiscountEndScreen from "../components/discount/discountEndScreen";
+
+import { freeRegistration } from "../utils/discounts/freeRegistration";
+import FreeRegisterDiscount from "@/components/discount/freeRegisterDiscount";
+import RegisterFree from "@/components/discount/registerFree";
+
+const FreeRegistration: NextPage = () => {
+  const [searchResult, setSearchResult] = useState<SearchResult | undefined>();
+  const [screen, setScreen] = useState<number>(1);
+
+  useEffect(() => {
+    const currentDate = new Date();
+    const timestamp = currentDate.getTime();
+
+    if (timestamp >= freeRegistration.expiry) setScreen(0);
+  }, []);
+
+  const goBack = () => setScreen(screen - 1);
+
+  return (
+    <div className={homeStyles.screen}>
+      {screen === 0 ? (
+        <DiscountEndScreen
+          title={`${freeRegistration.name} discount has ended`}
+          image={freeRegistration.image}
+        />
+      ) : null}
+      {screen === 1 ? (
+        <FreeRegisterDiscount
```

Comment:
I think we should harmonize this naming with the `DiscountRenewalScreen` we have on the `freerenewal.tsx` page.

Let's choose: `FreeRegisterPresentation` && `FreeRenewalPresentation`. 

Makes more sense like so imo !